app/helpers/interfaces_helper.rb

```diff
@@ -4,4 +4,8 @@ def interfaces_types
       [_(nic_type.humanized_name), nic_type]
     end
   end
+
+  def provided_mac?(nic)
+    nic&.host&.compute_resource&.provided_attributes&.try(:[], :mac)
```

Comment:
The `&` is ambigous.
```suggestion
    nic&.host&.compute_resource&.provided_attributes.try(:[], :mac)
```